lms/djangoapps/support/urls.py

```diff
@@ -5,19 +5,16 @@
 
 from lms.djangoapps.support.views.contact_us import ContactUsView
 from support.views.certificate import CertificatesSupportView
-from support.views.course_entitlements import EntitlementSupportView
 from support.views.enrollments import EnrollmentSupportListView, EnrollmentSupportView
 from support.views.index import index
 from support.views.manage_user import ManageUserDetailView, ManageUserSupportView
 from support.views.refund import RefundSupportView
 
-COURSE_ENTITLEMENTS_VIEW = EntitlementSupportView.as_view()
```

Comment:
@Mjloturco good point above. Should I add back this line and line 20 as well?

Reply:
Yup